sdcm/cluster_aws.py

```diff
@@ -445,7 +451,6 @@ def init(self):
                 self.allocate_and_attach_elastic_ip(self.parent_cluster, self.dc_idx)
                 resources_to_tag.append(self.eip_allocation_id)
             self._ec2_service.create_tags(Resources=resources_to_tag, Tags=tags_as_ec2_tags(self.tags))
-
```

Comment:
Is it intentional change? Looks like a mistake.

Reply:
yeah a mistake (touch this area first, thing to move the tag to the __init__)